python/lsst/ip/diffim/subtractImages.py

```diff
@@ -712,7 +743,12 @@ def _checkMask(mask, sources, badMaskPlanes):
             kept (True) or rejected (False) based on the value of the
             mask plane at its location.
         """
-        badPixelMask = lsst.afw.image.Mask.getPlaneBitMask(badMaskPlanes)
+        setBadMaskPlanes = [
+            maskPlane for maskPlane in badMaskPlanes if maskPlane in mask.getMaskPlaneDict().keys()
+        ]
+
+        badPixelMask = mask.getPlaneBitMask(setBadMaskPlanes)
```

Comment:
Thanks for changing this. We need to switch from `lsst.afw.image.Mask.getPlaneBitMask` to `mask.getPlaneBitMask` in general (see https://github.com/lsst/meas_algorithms/pull/329#discussion_r1195795073)